llvm/docs/IncidentResponseGuide.md

```diff
@@ -0,0 +1,87 @@
+============================
+LLVM Incident Response Guide
```

Comment:
My understanding is that this guide covers Incident Response, for a specific class of Incidents, but not all possible things people might think of as "incidents"?

I'm guessing some might be security incidents, others might be more "accidental outage" kind of incidents?
And this only relates to incidents related to project infrastructure, rather than e.g. a security issue discovered in one of the LLVM projects?

I'm wondering if a more accurate title is possible?
Maybe "LLVM Infrastructure Incident Response Guide"?